README.md

```diff
@@ -49,19 +49,15 @@ To enable it, you must either set the USE_REAL_AWS variable or specify a
 real openshift-ansible image to use in the ANSIBLE_IMAGE variable.
 
   * `USE_REAL_AWS=1 contrib/examples/create-cluster.sh`
-  * `ANSIBLE_IMAGE=openshift/origin-ansible:latest contrib/examples/create-cluster.sh`
+  * `ANSIBLE_IMAGE=cluster-operator-ansible:canary contrib/examples/create-cluster.sh`
 
 ## Developing With OpenShift Ansible
 
-You can build your own custom version of [https://github.com/openshift/openshift-ansible](OpenShift Ansible) when working on playbooks and roles from the openshift-ansible repo:
-
-  `docker build -f images/installer/Dockerfile -t dgoodwin-origin-ansible:latest .`
-
-You can then use this image with Cluster Operator via:
+The Cluster Operator uses it's own Ansible image which layers our playbooks and roles on top of the upstream (https://github.com/openshift/openshift-ansible)[OpenShift Ansible] images. Typically our Ansible changes only require work in this repo. See the `build/cluster-operator-ansible` directory for the Dockerfile and playbooks we layer in.
 
-  `ANSIBLE_IMAGE="dgoodwin-origin-ansible:latest" ANSIBLE_IMAGE_PULL_POLICY="Never" contrib/examples/create-cluster.sh`
+To build the *cluster-operator-ansible* image you can just run `make images` normally. **WARNING**: This image is built using OpenShift Ansible v3.9.
```

Comment:
You can specify CO_ANSIBLE_URL and CO_ANSIBLE_BRANCH to use a different branch/repository for the base openshift-ansible.